packages/state-manager/src/sagas/publicChannels/publicChannels.selectors.ts

```diff
@@ -73,6 +73,28 @@ export const currentChannelAddress = createSelector(
   }
 )
 
+export const dynamicSearchedChannels = (channelInput: string) =>
+  createSelector(publicChannels, publicChannelsSelector => {
+    const recentChannels = publicChannelsSelector
```

Comment:
I wish it was a separate selector. Try to make them as atomic as possible. See eg. `displayableCurrentChannelMessages` - it reuses `sortedCurrentChannelMessages` which reuses `currentChannelMessages` etc... thanks to that we get the best of memoization in selectors.